src/infrasys/time_series_metadata_store.py

```diff
@@ -122,48 +176,56 @@ def add(
                 None,  # auto-assigned by sqlite
                 str(metadata.time_series_uuid),
                 metadata.type,
+                time_series_category,
                 initial_time,
                 resolution,
+                horizon,
+                interval,
+                window_count,
+                metadata.length if hasattr(metadata, "length") else None,
                 metadata.variable_name,
                 str(owner.uuid),
                 owner.__class__.__name__,
-                attribute_hash,
-                json.dumps(serialize_value(metadata)),
+                "Component",
+                json.dumps(metadata.features),
```

Comment:
This is not compatible with IS.jl. We do this:
```Julia
function make_features_string(features::Dict{String, Union{Bool, Int, String}})
    key_names = sort!(collect(keys(features)))
    data = [Dict(k => features[k]) for k in key_names]
    return JSON3.write(data)
end

```

Reply:
is there a reason why we want this to be sorted?

Reply:
Oh I see now this is a json array. Is there a reason why we want it to be  a json array?

Reply:
Ok I added the same function now.

Reply:
Ok, @daniel-thom , I think we should not be saving the data like this: [{"model_year": "2030"}, {"scenario": "high"}] and save it instead like this: {"model_year": "2030", "scenario": "high"}. The latter allow us to create a good index for features if needed.